content/docs/rfcs/14/README.md

```diff
@@ -1,120 +1,102 @@
 
-Since Waku v2 is using libp2p PubSub, some of these fields can be dropped.
-The previous `topic` field corresponds to `contentTopic`.
-The previous `data` field corresponds to the `payload` field.
+## Reliability of the `ephemeral` attribute
 
-# Security Consideration
+The waku message `ephemeral` attribute is set by the sender. Since there is currently no incentive mechanism for network participants to behave correctly, this field is inherently insecure. A malicious actor can tamper with the value of a waku message’s `ephemeral` attribute, and the receiver would not be able to verify the integrity of the message.
 
-## Confidentiality, integrity, and authenticity 
-It is up to the application layer as to what level confidentiality, integrity and authenticity of the `payload` of `WakuMessage` matters. 
-Accordingly, the application layer shall utilize the encryption and signature schemes supported in WAKU2 to meet the application-specific privacy needs.
-The set of supported schemes in WAKU2 is presented in [WAKU2-PAYLOAD](/spec/26).
+## Copyright
 
-## Reliability of the WakuMessage timestamp
-
-The `timestamp` field in `WakuMessage` is set by the sender.
-Because `timestamp` isn't independently verified, this field is prone to exploit and misuse.
-It should not solely be relied upon for operations such as message ordering.
-
-For example, a malicious node can arbitrarily set the  `timestamp` of a `WakuMessage` to a high value so that it always shows up as the most recent message in a chat application.
-Applications using the `WakuMessage`'s `timestamp` field are recommended to use additional methods for more robust message ordering.
-An example of how to deal with message ordering against adversarial message timestamps can be found in the Status protocol, see [6/PAYLOADS](https://specs.status.im/spec/6#clock-vs-timestamp-and-message-ordering).
-
-## Reliability of the ephemeral flag
-
-The `ephemeral` field in `WakuMessage` is set by the sender.
-Since there is currently no incentive mechanism for nodes that implement [13/WAKU2-STORE](/spec/13) and [11/WAKU2-RELAY](/spec/11) to behave correctly, this field is inherently unsecure.
-Malicious nodes that implement [11/WAKU2-RELAY](/spec/11) can flip the value of the ephemeral flag, and nodes that receive such messages would have no mechanism to verify the integrity of the message.
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
 
-# Copyright
+# References
 
-Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+TBD
```

Comment:
I am unsure exactly what to put here 🤔

Reply:
I'd say the links you have already put in the text.

Reply:
It should be summarized links that are in text